File: base16/crosh/colors/base16-vice.js
// Base16 Vice Dark
// Scheme: Thomas Leon Highbaugh

var color_scheme = {
        'base00': '#181818',
        'base01': '#222222',
        'base02': '#323232',
        'base03': '#3f3f3f',
        'base04': '#666666',
        'base05': '#818181',
        'base06': '#c6c6c6',
        'base07': '#e9e9e9',
        'base08': '#ff29a8',
        'base09': '#85ffe0',
        'base0A': '#f0ffaa',
        'base0B': '#0badff',
        'base0C': '#8265ff',
        'base0D': '#00eaff',
        'base0E': '#00f6d9',
        'base0F': '#ff3d81',
};

term_.prefs_.set('background-color', color_scheme.base00);
term_.prefs_.set('foreground-color', color_scheme.base05);
term_.prefs_.set('cursor-color', "rgba(129, 129, 129, 0.5)");

term_.prefs_.set('color-palette-overrides', 
                        [color_scheme.base00,
                        color_scheme.base08,
                        color_scheme.base0B,
                        color_scheme.base0A,
                        color_scheme.base0D,
                        color_scheme.base0E,
                        color_scheme.base0C,
                        color_scheme.base05,
                        color_scheme.base03,
                        color_scheme.base08,
                        color_scheme.base0B,
                        color_scheme.base0A,
                        color_scheme.base0D,
                        color_scheme.base0E,
                        color_scheme.base0C,
                        color_scheme.base07,
                        color_scheme.base09,
                        color_scheme.base0F,
                        color_scheme.base01,
                        color_scheme.base02,
                        color_scheme.base04,
                        color_scheme.base06]);

File: base16/pygments/pygments_base16/base16-vice.py
from pygments.style import Style
from pygments.token import (
    Comment, Error, Keyword, Literal, Name, Number, Operator, String, Text
)


class Base16Style(Style):
    base00 = '#17191E'
    base01 = '#22262d'
    base02 = '#3c3f4c'
    base03 = '#383a47'
    base04 = '#555e70'
    base05 = '#8b9cbe'
    base06 = '#B2BFD9'
    base07 = '#f4f4f7'
    base08 = '#ff29a8'
    base09 = '#85ffe0'
    base0a = '#f0ffaa'
    base0b = '#0badff'
    base0c = '#8265ff'
    base0d = '#00eaff'
    base0e = '#00f6d9'
    base0f = '#ff3d81'

    default_style = ''

    background_color = base00
    highlight_color = base02

    styles = {
        Text: base05,
        Error: base08,  # .err

        Comment: base03,  # .c
        Comment.Preproc: base0f,  # .cp
        Comment.PreprocFile: base0b,  # .cpf

        Keyword: base0e,  # .k
        Keyword.Type: base08,  # .kt

        Name.Attribute: base0d,  # .na
        Name.Builtin: base0d,  # .nb
        Name.Builtin.Pseudo: base08,  # .bp
        Name.Class: base0d,  # .nc
        Name.Constant: base09,  # .no
        Name.Decorator: base09,  # .nd
        Name.Function: base0d,  # .nf
        Name.Namespace: base0d,  # .nn
        Name.Tag: base0e,  # .nt
        Name.Variable: base0d,  # .nv
        Name.Variable.Instance: base08,  # .vi

        Number: base09,  # .m

        Operator: base0c,  # .o
        Operator.Word: base0e,  # .ow

        Literal: base0b,  # .l

        String: base0b,  # .s
        String.Interpol: base0f,  # .si
        String.Regex: base0c,  # .sr
        String.Symbol: base09,  # .ss
    }


from string import capwords  # noqa: E402
Base16Style.__name__ = 'Base16{}Style'.format(
    capwords('vice', '-').replace('-', '')
)
globals()[Base16Style.__name__] = globals()['Base16Style']
del globals()['Base16Style']
del capwords
